src/lib/y2storage/storage_manager.rb

```diff
@@ -164,25 +190,6 @@ def commit
       storage.commit
```

Comment:
Out of scope, but maybe good to add a FIXME since we are touching all this: we need to add proper CommitCallbacks to yast2-storage-ng